src/server_manager/cloud/digitalocean_api.ts

```diff
@@ -220,6 +220,8 @@ class RestApiSession implements DigitalOceanSession {
           // this.response may be empty.
           const responseObj = (xhr.response ? JSON.parse(xhr.response) : {});
           resolve(responseObj);
+        } else if (xhr.status === 401) {
+          reject(new XhrError());
```

Comment:
The handling below seems more useful,  since it raises an error with more information.